internal/infrastructure/http/handlers/handlers.go

```
@@ -0,0 +1,139 @@
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		return
+	}
+
+	metrics, err := h.MetricsStore.GetAllMetrics()
+	if err != nil {
+		http.Error(w, http.StatusText(http.StatusInternalServerError), http.StatusInternalServerError)
+		return
+	}
+
+	w.WriteHeader(200)
+	tmpl.Execute(w, metrics)
```

golangci-lint (1.62.2)

Error return value of `tmpl.Execute` is not checked

(errcheck)